portal-impl/src/com/liferay/portal/util/PortalImpl.java

```diff
@@ -5665,7 +5665,12 @@ public String getUserName(
 
 		if (user != null) {
 			if (userAttribute.equals(UserAttributes.USER_NAME_FULL)) {
-				userName = user.getFullName();
+				if ((user.getFullName() != null) ||
+					!user.getFullName(
```

Comment:
@MirnaGama Chaining is basically avoided in Liferay, and `==null` + `isEmpty` check can be simplified by using `Validator.isBlank()`.

So, please do something like

```
String userFullName = user.getFullName();

if (Validator.isNotBlank(userFullName)) {
    userName = userFullName;
}
```

Reply:
Done ! 

https://github.com/liferay-core-infra/liferay-portal/pull/5534/commits/dc136323cc28ed9f86c17f5fb21e1ff3276a1f94